notebooks/Imputation_best_practices/Imputation_best_practices.ipynb

```diff
@@ -0,0 +1,4557 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    print(f"The mean of 1% is {m1} and varience 1% is {var_res1}")
</pre><p>Here we're computing mean of the absolute diff i.e. abs(imputed_value_i - actual_value_i) right? Wouldn't this number change with the scale of the actual values? Like if the mean absolute difference is 0.1 and the original data is in the range of 1000-5000, then this would be a considered a great approach, but if the original data is in the range of 0-1, then would this still be a great approach?</p><p> </p><p>So what do think about using mean absolute percent error instead? i.e. mean of abs(imputed_value_i - actual_value_i) / actual_value_i</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Not required, the idea is to highlight the best practices and hence in the second dataset example both scaled and non-scaled version of the data has been used to show that if we use scaling before performing the imputations then the result will be much better (the purpose of the notebook). Hope this approach is fine!</p>